packages/server/src/internals/server/kratos-middleware.ts

```diff
@@ -111,8 +101,15 @@ function createKratosRegisterHandler(kratos: V0alpha2Api): RequestHandler {
         errorContext: { userId, error },
       })
 
-      res.statusCode = 500
-      return res.end('Internal error in after hook')
+      response.statusCode = 500
+      return response.end('Internal error in after hook')
     }
   }) as RequestHandler
+
+  function assertBodyContainsUserId(body: unknown): body is { userId: string } {
```

Comment:
```suggestion
  function containsUserId(body: unknown): body is { userId: string } {
```

By the way I do not see a need for this function since you can also write ` t.type({ userId: t.string }).is(body)` instead